requirements.txt

```diff
@@ -0,0 +1,30 @@
+appdirs==1.4.3
+attrs==19.3.0
+certifi==2019.11.28
+chardet==3.0.4
+ConfigArgParse==1.1
+datrie==0.8
+decorator==4.4.2
+docutils==0.16
+gitdb==4.0.2
+GitPython==3.1.0
+idna==2.9
+importlib-metadata==1.5.0
+ipython-genutils==0.2.0
+jsonschema==3.2.0
+jupyter-core==4.6.3
+nbformat==5.0.4
+pkg-resources==0.0.0
+psutil==5.7.0
+pyrsistent==0.15.7
+PyYAML==5.3
+ratelimiter==1.2.0.post0
+requests==2.23.0
+six==1.14.0
+smmap==3.0.1
+snakemake==5.10.0
+toposort==1.5
+traitlets==4.3.3
+urllib3==1.25.8
+wrapt==1.12.1
+zipp==3.1.0
```

Comment:
Did you mean to add this to the commit?

Reply:
Not exactly this, but we need some dependencies management (for test purposes only). On my system, I switched to an environment for an earlier snakemake version, and either `snakemake.utils` or `read_job_properties` from the `snakemake.utils` package did not exist. This is also important to make the tests reproducible (can change this to `requirements_tests.txt` if you prefer). Tell me if you think this is needed, or if you prefer to do this in other way.

Reply:
Sure. `pyproject.toml` is the preferred method for describing these things now. You can merge this file as is but I will move it into a `pyproject.toml` file.